From 2fb265da041261ee22bccd61d2570511bee1e251 Mon Sep 17 00:00:00 2001
From: Reuben Steenekamp <reuben.steenekamp@gmail.com>
Date: Tue, 18 Jan 2022 15:39:58 +0200
Subject: [PATCH] Makefile changes

---
 Makefile | 30 +++---------------------------
 1 file changed, 3 insertions(+), 27 deletions(-)

diff --git a/Makefile b/Makefile
index bcadf09..223a288 100644
--- a/Makefile
+++ b/Makefile
@@ -30,38 +30,14 @@ VOSK_HEADERS= \
 	spk_model.h \
 	vosk_api.h
 
-CFLAGS=-g -O3 -std=c++17 -fPIC -DFST_NO_DYNAMIC_LINKING \
-	-I. -I$(KALDI_ROOT)/src -I$(OPENFST_ROOT)/include $(EXTRA_CFLAGS) 
+CFLAGS=-g -O3 -std=c++17 -fPIC \
+	-I. -I$(KALDI_ROOT)/include/kaldi -I$(OPENFST_ROOT)/include $(EXTRA_CFLAGS)
 
-LIBS= \
-	$(KALDI_ROOT)/src/online2/kaldi-online2.a \
-	$(KALDI_ROOT)/src/decoder/kaldi-decoder.a \
-	$(KALDI_ROOT)/src/ivector/kaldi-ivector.a \
-	$(KALDI_ROOT)/src/gmm/kaldi-gmm.a \
-	$(KALDI_ROOT)/src/nnet3/kaldi-nnet3.a \
-	$(KALDI_ROOT)/src/tree/kaldi-tree.a \
-	$(KALDI_ROOT)/src/feat/kaldi-feat.a \
-	$(KALDI_ROOT)/src/lat/kaldi-lat.a \
-	$(KALDI_ROOT)/src/lm/kaldi-lm.a \
-	$(KALDI_ROOT)/src/rnnlm/kaldi-rnnlm.a \
-	$(KALDI_ROOT)/src/hmm/kaldi-hmm.a \
-	$(KALDI_ROOT)/src/transform/kaldi-transform.a \
-	$(KALDI_ROOT)/src/cudamatrix/kaldi-cudamatrix.a \
-	$(KALDI_ROOT)/src/matrix/kaldi-matrix.a \
-	$(KALDI_ROOT)/src/fstext/kaldi-fstext.a \
-	$(KALDI_ROOT)/src/util/kaldi-util.a \
-	$(KALDI_ROOT)/src/base/kaldi-base.a \
-	$(OPENFST_ROOT)/lib/libfst.a \
-	$(OPENFST_ROOT)/lib/libfstngram.a
+LIBS=
 
 
 ifeq ($(HAVE_OPENBLAS_CLAPACK), 1)
     CFLAGS += -I$(OPENBLAS_ROOT)/include
-    LIBS += \
-        $(OPENBLAS_ROOT)/lib/libopenblas.a \
-        $(OPENBLAS_ROOT)/lib/liblapack.a \
-        $(OPENBLAS_ROOT)/lib/libblas.a \
-        $(OPENBLAS_ROOT)/lib/libf2c.a
 endif
 
 ifeq ($(HAVE_MKL), 1)
-- 
2.34.1

